### DISCUSSION TITLE
2.7 & Speedo

### DISCUSSION THREAD

**Discussion by doc1011** [CircuitSetup/Time-Circuits-Display#38] [General]
Hey folks, I saw reference to 2.7 with the every second beep being introduced. Any idea as to when a release .bin to update via the web will feature here? Latest version on the releases page is 2.6. 

Also wondering if anyone has seen any updates on the speedo from CircuitSetup?

Cheers all

**Comment by CircuitSetup** [comment]
v2.7 is still being tested, with a tentative release of mid next week.

The speedo PCB will be offered first (with and without GPS), and possibly a pre-cut box, wire connector, and wire. The PCB's are still being finalized, so there is currently no ETA.

**Reply by doc1011** [reply]
Awesome, cheers mate!
